src/gatsby/on-render-body.js

```diff
@@ -48,29 +65,38 @@ export default ({ setHeadComponents }, pluginOptions) => {
     // TO DO - Warn about missing options
   }
 
-  let agent;
-  if (instrumentationType === 'lite') {
-    agent = liteAgent;
-  }
-
-  if (instrumentationType === 'pro') {
-    agent = proAgent;
-  }
+  const init = `
+    window.NREUM||(NREUM={});
+    NREUM.init={distributed_tracing:{enabled:false},privacy:{cookies_enabled:true}};
```

Comment:
It'd be nice to make these configurable - some companies use different values for these settings. (unless I'm mistaken and these features are exclusively toggled in the application settings and don't require client side config updates)

Reply:
@passion4code Agreed, let's make this configurable, I'll add that